apps/challenges/models.py

```diff
@@ -98,7 +118,7 @@ class ChallengePhase(TimeStampedModel):
     challenge = models.ForeignKey('Challenge')
     is_public = models.BooleanField(default=False)
     is_submission_public = models.BooleanField(default=False)
-    test_annotation = models.FileField(upload_to=RandomFileName("test_annotations"))
+    test_annotation = models.FileField(upload_to=RandomFileName("test_annotations"), default=False)
```

Comment:
@TroJan I added it as I am checking the condition of first time upload of a `test_annotation`. Please refer [here](https://github.com/Cloud-CV/EvalAI/pull/912/files#diff-06139830cb2f143920edd3015bb7b1baR153).
Can I create a new issue for it ?